build/build.xml

```diff
@@ -287,6 +288,54 @@
 			</else>
 		</if>
 
+		<!-- Checkout latest viglink to ext folder -->
+		<available file="${dir}/ext" type="dir" property="add-viglink-ext" />
+		<exec dir="${dir}"
+			command='php -r "echo version_compare(&apos;${version}&apos;, &apos;3.2.0-RC2&apos;, &apos;>=&apos;) ? &apos;true&apos; : &apos;false&apos;;"'
+			checkreturn="true"
+			outputProperty='viglink-available' />
+		<if>
+			<and>
+				<equals arg1="${add-viglink-ext}" arg2="1" trim="true" />
+				<or>
+					<equals arg1="${revision}" arg2="HEAD" trim="true" />
+					<equals arg1="${viglink-available}" arg2="1" trim="true" />
+				</or>
+			</and>
+			<then>
+				<exec dir="${dir}/ext" command="mkdir phpbb" passthru="true" />
+
+				<exec dir="${dir}/ext/phpbb" command="git clone https://github.com/phpbb-extensions/viglink.git viglink" passthru="true" checkreturn="true" />
+				<if>
+					<equals arg1="${revision}" arg2="HEAD" trim="true" />
+					<then>
+						<exec dir="${dir}/ext/phpbb/viglink"
+							command="git rev-parse release-phpbb-${version}"
+							returnProperty='viglink_head_tag_exists' />
+						<if>
+							<equals arg1="${viglink_head_tag_exists}" arg2="0" trim="true" />
+							<then>
+								<exec dir="${dir}/ext/phpbb/viglink" command="git checkout release-phpbb-${version}" passthru="true" />
+							</then>
+							<else>
+								<exec dir="${dir}/ext/phpbb/viglink" command="git checkout master" passthru="true" />
+							</else>
+						</if>
+					</then>
+					<else>
+						<exec dir="${dir}/ext/phpbb/viglink" command="git checkout release-phpbb-${version}" passthru="true" />
+					</else>
+				</if>
+				<delete dir="${dir}/ext/phpbb/viglink/.git" />
+				<delete dir="${dir}/ext/phpbb/viglink/tests" />
+				<delete dir="${dir}/ext/phpbb/viglink/travis" />
+				<delete file="${dir}/ext/phpbb/viglink/.gitattributes" />
+				<delete file="${dir}/ext/phpbb/viglink/.travis.yml" />
+				<delete file="${dir}/ext/phpbb/viglink/phpunit.xml.dist" />
+				<delete file="${dir}/ext/phpbb/viglink/README.md" />
```

Comment:
We should think about adding a .gitattributes to the viglink repo, so we can remove this. ;)